frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -206,14 +216,28 @@ class Chart extends Component<Props, State> {
   };
 
   private onResize = () => {
+    this.checkIfMobile();
     if (this.resizeTimerID) {
       clearTimeout(this.resizeTimerID);
     }
     this.resizeTimerID = setTimeout(() => {
       if (!this.chart || !this.ref.current) {
         return;
       }
-      this.chart.resize(this.ref.current.offsetWidth, this.height);
+      this.chart.resize(!this.state.isMobile ? this.ref.current.offsetWidth : window.outerWidth, !this.state.isMobile ? this.height : this.mobileHeight);
```

Comment:
this is a bit hard to read, could you save the const first in width and height varibables?